packages/react-native/ReactAndroid/src/main/java/com/facebook/react/turbomodule/core/TurboModuleManager.java

```diff
@@ -249,32 +249,63 @@ private NativeModule getOrCreateModule(
     }
   }
 
-  /** Which TurboModules have been created? */
+  @Deprecated
   public Collection<TurboModule> getModules() {
```

Comment:
Wouldn't it be nice if we have deprecation _statement_ here as well ?
Just being curious since this is marked **_public_**